### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: mqtt-ssl/build.gradle [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
+plugins {
+    id("io.micronaut.build.internal.module")
+}
 dependencies {
-    annotationProcessor "io.micronaut:micronaut-inject-java"
-    annotationProcessor "io.micronaut.docs:micronaut-docs-asciidoc-config-props:$micronautDocsVersion"
     implementation("io.micronaut:micronaut-http")
     api project(":mqtt-core")
     implementation "org.bouncycastle:bcpkix-jdk15on:1.70"
     api "org.testcontainers:spock:1.16.3"
```
Comment (timyates) [inline mqtt-ssl/build.gradle:8]: I don't believe this should be an `api` dependency 🤔

**Review by timyates — APPROVED** [review]
👍

**Comment by timyates** [comment]
Spotted the callouts were wrong for one of the snippets in the docs, pushed the fix here as otherwise it would be a horrible merge

File: mqtt-ssl/build.gradle (hunk 1 above)
Comment (sdelamo) [reply mqtt-ssl/build.gradle:8]: test containers one. Yeah, I found it surprising as well. I did not want to touch it in this PR.  I think I am going to change it in a different Pr.

Comment (sdelamo) [reply mqtt-ssl/build.gradle:8]: I fixed it here: https://github.com/micronaut-projects/micronaut-mqtt/commit/0a187d12386a4736a82af300efb7b90423e49a7b

**Pull request merged**